### PR TITLE
fixed types for mapbox/wellknown

### PR DESCRIPTION
**Pull request by davide-scalzo**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: Wellknown should parse / stringify GeoJSON Geometry, link to GeoJSON spec here https://tools.ietf.org/html/rfc7946#section-3.1

### CODE REVIEW

File: types/wellknown/index.d.ts [hunk 1]
```diff
@@ -1,7 +1,32 @@
 // Type definitions for wellknown 0.5
 // Project: https://github.com/mapbox/wellknown#readme
-// Definitions by: Yair Tawil <https://github.com/yairtawil>
+// Definitions by: Davide Scalzo <https://github.com/davodesign84>
```
Comment (peterblazejewicz) [inline types/wellknown/index.d.ts]: Is that indended change? Would adding a new author on the next line suffice?

File: types/wellknown/index.d.ts [hunk 2]
```diff
@@ -1,7 +1,32 @@
...
+
+export type GeoJSONGeometry =
+    | GeoJSONPoint
+    | GeoJSONMultiPoint
+    | GeoJSONLineString
+    | GeoJSONMultiLineString
+    | GeoJSONPolygon
+    | GeoJSONMultiPolygon;
+
+export function parse(input: string): GeoJSONGeometry;
```
Comment (peterblazejewicz) [inline types/wellknown/index.d.ts]: So the documentation states it also returns `null`, when parse fails:
https://github.com/mapbox/wellknown#parsewkt

File: types/wellknown/wellknown-tests.ts [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import * as wellknown from 'wellknown';
 wellknown.parse("POINT(1 2)");
```
Comment (peterblazejewicz) [inline types/wellknown/wellknown-tests.ts]: This one should be probably updated to:
```ts
wellknown.parse("POINT(1 2)"); // $ExpectType wellknown.GeoJSONGeometry | null
```

File: types/wellknown/index.d.ts (hunk 1 above)
Comment (davodesign84) [reply types/wellknown/index.d.ts]: yup, good point

File: types/wellknown/index.d.ts (hunk 2 above)
Comment (davodesign84) [reply types/wellknown/index.d.ts]: ah great spot!

File: types/wellknown/wellknown-tests.ts (hunk 3 above)
Comment (davodesign84) [reply types/wellknown/wellknown-tests.ts]: For some reason this fails with
```
ERROR: 4:1  expect  TypeScript@next expected type to be:
  GeoJSONGeometry | null
got:
  GeoJSONPoint | GeoJSONMultiPoint | GeoJSONLineString | GeoJSONMultiLineString | GeoJSONPolygon | GeoJSONMultiPolygon | null
```

Comment (davodesign84) [reply types/wellknown/wellknown-tests.ts]: Turns out it seems like $ExpectType her only recognizes a single type..

Comment (peterblazejewicz) [reply types/wellknown/wellknown-tests.ts]: Sorry for the fuss here, I was wrong (looked probably into wrong version). You're correctly on this `GeoJSONGeometry` is OK

**Review by peterblazejewicz — COMMENTED** [review]
/cc @yairtawil

**Pull request merged**